### PR TITLE
Subgroup switch

### PR DESCRIPTION
**Pull request by AyeRlS**
Adding the switch "group only / all subgroups", as demanded in issue #79.

### CODE REVIEW

**Review by smadbe — CHANGES_REQUESTED** [review]
👍 
Few details to be changed.

File: src/app/modules/group/components/pending-request/pending-request.component.ts [hunk 1]
```diff
@@ -178,4 +183,10 @@ export class PendingRequestComponent implements OnInit, OnChanges {
       this.reloadData();
     }
   }
+
+  onSwitchChange(isAllowed: number)
```
[the remark this answers is not in the document]
Comment (AyeRlS) [reply src/app/modules/group/components/pending-request/pending-request.component.ts]: implemented !

File: src/app/modules/group/components/pending-request/pending-request.component.ts [hunk 2]
```diff
@@ -49,10 +49,15 @@ export class PendingRequestComponent implements OnInit, OnChanges {
     { field: 'user.login', header: 'USER' },
     { field: 'at', header: 'REQUESTED ON' },
   ];
+  switchItems= [
+    {label:'This group only'},
+    {label:'All subgroups'}
+  ];
```
[the remark this answers is not in the document]
Comment (AyeRlS) [reply src/app/modules/group/components/pending-request/pending-request.component.ts]: done

[the remark this answers is not in the document]
Comment (smadbe) [reply src/app/modules/group/components/pending-request/pending-request.component.ts]: When there is a suggestion, you can just "commit suggestion" as well

**Review by smadbe — APPROVED** [review]

**Pull request merged**